### PR TITLE
LOC CHECKIN | dotnet/roslyn  release/dev16.4-vs-deps | 20191101

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/CodeStyle/Core/CodeFixes/xlf/CodeStyleFixesResources.zh-Hans.xlf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hans" original="../CodeStyleFixesResources.resx">
```
Comment (JoeRobich) [inline src/CodeStyle/Core/CodeFixes/xlf/CodeStyleFixesResources.zh-Hans.xlf]: Is this a necessary change? or does it matter?

Comment (janbaltus) [reply src/CodeStyle/Core/CodeFixes/xlf/CodeStyleFixesResources.zh-Hans.xlf]: @JoeRobich  we will update PR within an hour

**Comment by JoeRobich** [comment]
FYI @jinujoseph

File: src/CodeStyle/Core/CodeFixes/xlf/CodeStyleFixesResources.zh-Hans.xlf (hunk 1 above)
Comment (janbaltus) [reply src/CodeStyle/Core/CodeFixes/xlf/CodeStyleFixesResources.zh-Hans.xlf]: @JoeRobich PR is updated with correct language code

**Pull request merged**